JavaScript/README.md

```diff
@@ -19,9 +19,10 @@
 
 ## Searching
 
-- [Linear Search](./search/linear_search.js)
 - [Binary Search](./search/binary_search.js)
+- [Linear Search](./search/linear_search.js)
```

Comment:
try not to change the existing sequence.

Reply:
Ok Understood :+1: